### PR TITLE
[nan-621] output # of records

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Ouput a summary of records that would be added, updated, or deleted by the integration script

## Issue ticket number and link
NAN-621

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/cli/lib/services/dryrun.service.ts [hunk 1]
```diff
@@ -143,7 +143,10 @@ class DryRunService {
             stubbedMetadata = rawStubbedMetadata;
         }
 
-        const logMessages: string[] = [];
+        const logMessages = {
+            counts: { updated: 0, added: 0, deleted: 0 },
```
Comment (bodinsamuel) [inline packages/cli/lib/services/dryrun.service.ts:147]: not sure I would have mixed those things, they seems a bit unrelated?

**Review by bodinsamuel — APPROVED** [review]

File: packages/cli/lib/services/dryrun.service.ts (hunk 1 above)
Comment (khaliqgant) [reply packages/cli/lib/services/dryrun.service.ts:147]: Just oganized this way to make the view concise

image
